eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -169,6 +169,15 @@ function DeployStressPackage(
 
         Run docker build -t $imageTag -f $dockerFile $dockerBuildFolder
         if ($LASTEXITCODE) { return }
+
+        Write-Host "`nContainer image '$imageTag' successfully built. To run commands on the container locally:" -ForegroundColor Blue
+        Write-Host "  docker run -it $imageTag" -ForegroundColor DarkBlue
```

Comment:
This is practically invisible on a black display. How about just plain White text?

Reply:
(by which I mean - no explicit color)